Dockerfile.example

```diff
@@ -0,0 +1,50 @@
+##
+# Dockerfile for analytics-reporter.
+# https://github.com/18F/analytics-reporter
+#
+# For information about setting up keys, etc, see README.md.
+# This Dockerfile requires keys, and optional customizations, to be set below.
+##
+
+FROM debian:jessie
+RUN apt-get update && apt-get install -yq \
+  nodejs-legacy \
+  npm
+
+##
+# Install analytics-reporter.
+##
+RUN npm install -g analytics-reporter
+
+##
+# Add pem key. Requires the pem key to be named secret_key.pem and present
+# in the same directory.
+##
+RUN mkdir /opt/analytics
+COPY ./secret_key.pem /opt/analytics/secret_key.pem
+ENV ANALYTICS_KEY_PATH="/opt/analytics/secret_key.pem"
+##
+# These two must be changed!
+##
+ENV ANALYTICS_REPORT_IDS="ga:XXXXXX"
```

Comment:
Can't these be stored outside of the Dockerfile...? Feels like it's asking for trouble for people to add private information to a versioned file.